tsa-core/src/main/kotlin/org/usvm/machine/state/TvmSliceUtils.kt

```diff
@@ -1488,6 +1500,13 @@ fun TvmState.readSliceDataPos(slice: UHeapRef) = fieldManagers.cellDataLengthFie
 
 fun TvmState.readSliceCell(slice: UHeapRef) = memory.readField(slice, sliceCellField, ctx.addressSort)
 
+@Suppress("Unused")
+fun TvmState.readCellData(cell: UHeapRef): UExpr<TvmCellDataSort> =
+    this.fieldManagers.cellDataFieldManager.readCellDataWithoutAsserts(this, cell)
```

Comment:
I don't like having this function. `readCellDataWithoutAsserts` may be dangerous, this is why is has such a long name